drivers/display/display_st7789v.c

```diff
@@ -88,6 +92,14 @@ static void st7789v_transmit(struct st7789v_data *data, u8_t cmd,
 	}
 }
 
+#ifdef CONFIG_DEVICE_POWER_MANAGEMENT
+static void st7789v_enter_sleep(struct st7789v_data *data)
+{
+	st7789v_transmit(data, ST7789V_CMD_SLEEP_IN, NULL, 0);
+	k_sleep(K_MSEC(5));
```

Comment:
Why the driver have to wait here for 5 ms?

Reply:
Thanks! I had another look at the data sheet and the wait should not be necessary. I'll remove it.